app/commands/solution/search_community_solutions.rb

```diff
@@ -10,11 +10,15 @@ def self.default_per
       DEFAULT_PER
     end
 
-    def initialize(exercise, page: nil, per: nil, criteria: nil)
+    def initialize(exercise, page: nil, per: nil, order: nil, criteria: nil, tests_status: nil, mentoring_status: nil, up_to_date: nil)
```

Comment:
I'd probably break this over multiple lines at this stage, with some logical grouping. 

Probably exercise,page,per,order, then the rest on the next line